subsys/net/lib/lwm2m/lwm2m_shell.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#define LOG_MODULE_NAME net_lwm2m_shell
+#define LOG_LEVEL CONFIG_LWM2M_LOG_LEVEL
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#include <stddef.h>
+#include <stdlib.h>
+
+#include <kernel.h>
+#include <net/lwm2m.h>
+#include <shell/shell.h>
+
+#define LWM2M_HELP_CMD "LwM2M commands"
+#define LWM2M_HELP_SEND "LwM2M SEND operation\nsend [OPTION]... [PATH]...\n-n\tSend as non-confirmable\nPaths are inserted without leading '/'"
+
+#define PR(fmt, ...) shell_fprintf(shell, SHELL_NORMAL, fmt, ##__VA_ARGS__)
+
+#define PR_SHELL(shell, fmt, ...) shell_fprintf(shell, SHELL_NORMAL, fmt, ##__VA_ARGS__)
+
+#define PR_ERROR(fmt, ...) shell_fprintf(shell, SHELL_ERROR, fmt, ##__VA_ARGS__)
+
+#define PR_INFO(fmt, ...) shell_fprintf(shell, SHELL_INFO, fmt, ##__VA_ARGS__)
+
+#define PR_WARNING(fmt, ...) shell_fprintf(shell, SHELL_WARNING, fmt, ##__VA_ARGS__)
```

Comment:
I think these macros are unnecessary.
`shell.h` defines `shell_error()`, `shell_info()`, `shell_warn()` and `shell_print()` so use those to get shorter calls.

Reply:
Fixed.